zerver/webhooks/trello/view/card_actions.py

```diff
@@ -169,6 +181,21 @@ def get_changed_due_date_body(payload, action_type):
     }
     return fill_appropriate_message_content(payload, action_type, data)
 
+def get_managed_desc_body(payload, action_type):
+    # type: (Mapping[str, Any], Text) -> Text
+    data = {
+        'desc': prettify_date(get_action_data(payload)['card'].get('desc'))
```

Comment:
Are you sure you want that? All other methods uses this syntax, like:

    def get_managed_due_date_body(payload, action_type):
        # type: (Mapping[str, Any], Text) -> Text
        data = {
            'due_date': prettify_date(get_action_data(payload)['card'].get('due'))
        }
        return fill_appropriate_message_content(payload, action_type, data)

(valid for both review comment)